packages/compiler/core/checker.ts

```diff
@@ -1182,9 +1183,8 @@ export function createChecker(program: Program): Checker {
     opReference: TypeReferenceNode | undefined
   ): OperationType | undefined {
     if (!opReference) return undefined;
-
     // Ensure that we don't end up with a circular reference to the same operation
-    const opSymId = operation.symbol ? getNodeSymId(operation) : undefined;
+    const opSymId = getNodeSymId(operation);
```

Comment:
Oh, I guess this is moved into getNodeSymId. I could see myself getting confused why the if below is needed now.